### PR TITLE
Leaf-3198 step 1 arrange classes in class folders

### PR DESCRIPTION
**Pull request by jampaul3**
Step 1 was to get all the class files into class folders, rather than have them placed willy nilly. All classes are now in sources and in the api folders. I didn't really change the api and controller folders. In moving the files I had to make sure they were being included correctly as well. So all files that uses a moved class file has changed as well.

### CODE REVIEW

**Review by maxrichardjr — APPROVED** [review]
I realize it was largely changes to path for mySQL and config.php, but it was a ton of changes, 83 files!!

File: LEAF_Request_Portal/templates/print_form.tpl [hunk 1]
```diff
@@ -153,6 +153,7 @@ function doSubmit(recordID) {
 		data: {CSRFToken: '<!--{$CSRFToken}-->'},
 		success: function(response) {
             if(response.errors.length == 0) {
+                console.log('it was a success with NO errors');
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/templates/print_form.tpl:156]: Should not need logs here for production.

File: LEAF_Request_Portal/templates/print_form.tpl [hunk 2]
```diff
@@ -161,6 +162,7 @@ function doSubmit(recordID) {
                 workflow.getWorkflow(recordID);
             }
             else {
+                console.log('it was a succes with errors');
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/templates/print_form.tpl:165]: Here too.

**Review by raphaelfretter — CHANGES_REQUESTED** [review]

**Review by shaneodd — APPROVED** [review]

File: LEAF_Request_Portal/templates/print_form.tpl (hunk 2 above)
Comment (jampaul3) [reply LEAF_Request_Portal/templates/print_form.tpl]: removed

**Review by raphaelfretter — APPROVED** [review]

**Review by aerinkayne — APPROVED** [review]

**Pull request closed without merging**